template/jgroups/udp.xml

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Would be good to add license headers.

Reply:
I don't know about license headers much? Should it contain xwiki license headers? As I have taken the file from JGroups ->
https://github.com/belaban/JGroups/blob/master/conf/udp.xml

Reply:
hmm I thought I remembered that we were providing this xml file somewhere on xwiki.org but cannot find it. In any case we need a license header for all files we provide. We have 2 solutions:
1) extract the default config file from the jgroups jar at execution time (on first run for example). This allows to be up to date and reduce maintenance cost.
2) provide the file as you did and put our own license headers at the top. See https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HForXMLfiles

I prefer option 1.

@tmortagne any opinion on this?

Reply:
This configuration file is not exactly the same with the jgroups jar.  I have changed it for docker as documented in https://github.com/belaban/jgroups-docker. So I think option 2 would be better